shuup/core/pricing/_priceful.py

```diff
@@ -158,6 +159,10 @@ def unit_discount_amount(self):
 
     @property
     def tax_rate(self):
+        return ensure_decimal_places(self.raw_tax_rate)
```

Comment:
Showing the values should done with formatting functions anyway. You don't want to show "0.123456789", but "12.346%" on "12,346 %" (depending on locale). So the rounding should happen in `format_percent` or the `|percent` Jinja-filter, not in Priceful which should just do the pure numbers without rounding.

Same applies for prices and other money amounts. We don't want to show the pure decimal value 12.4559, but rather "$12.46" or "12,46 $" etc. For money amounts there is `format_money` Python function and `|money` Jinja filter which should do the rounding.

I don't see any good reason to do rounding in Priceful level.